processor/src/main/java/org/mapstruct/ap/internal/model/beanmapping/SourceReference.java

```diff
@@ -206,12 +223,7 @@ private Parameter fetchMatchingParameterFromFirstSegment(String[] segments ) {
                 if ( parameter == null ) {
                     reportMappingError(
                         Message.PROPERTYMAPPING_INVALID_PARAMETER_NAME,
-                        parameterName,
-                        Strings.join(
-                            method.getSourceParameters(),
-                            ", ",
-                            Parameter::getName
-                        )
+                        parameterName, Strings.join( method.getSourceParameters(), ", ", Parameter::getName )
```

Comment:
The formatting is weird here. There are only whitespace changes here

Reply:
I adapter it and brought the Strings.join to the next line.. Better this way?